src/components/NavigationBar/index.tsx

```diff
@@ -53,26 +53,7 @@ const NavigationBar: React.FC = () => {
         </ul>
       </div>
 
-      {!menuState && (
-        <div className="navbar-mobile-div">
-          <div className="navbar-mobile">
-            <Link href="/">
-              <a>
-                <img src={ACMLogo.src} alt="ACM Logo" />
-                <p>at UCSD</p>
-              </a>
-            </Link>
-            <button className="icon" onClick={toggleMenu} type="button">
-              <img
-                className="menuicon"
-                src={MenuButtonIcon.src}
-                alt="Menu Icon"
-              />
-            </button>
-          </div>
-        </div>
-      )}
-      {menuState && (
+      {menuState ? (
```

Comment:
I agree with using a ternary in this situation where we choose between two options but for the other spots with conditional rendering, I'm personally still a fan of short-circuiting for conciseness and clarity because we can say `if (condition_met) { show component }` without having to write `else { show nothing }` when the condition fails. I understand the potential bug rendering the first value when short circuiting on an integer but so far we've been maintaining boolean conditions only so that shouldn't arise if the style is maintained.

Reply:
agree it's good to keep things concise, imo relying on developers to make sure only boolean conditions are used with conditional rendering is tenuous and unreliable though, and if we have a way to avoid that we should do so. in any case `!cond : null ( <jsx /> )` was done precisely so it remains quite concise, and otherwise maintains very similar shape/structure to `cond && ( <jsx /> )`